numpy/matrixlib/tests/test_defmatrix.py

```diff
@@ -322,9 +322,8 @@ def test_basic(self):
         assert_equal(x, [[0, 1], [0, 0], [0, 0]])
 
 
-class TestNewScalarIndexing(TestCase):
-    def setUp(self):
-        self.a = matrix([[1, 2], [3, 4]])
+class TestNewScalarIndexing(object):
+    a = matrix([[1, 2], [3, 4]])
```

Comment:
Same about setup function being called often. But again, since the test run through successfully, I am not worried. (if we get random failures here at some point, we can remember).